Kickstarter-iOS/Views/Controllers/CreatePasswordViewControllerTests.swift

```diff
@@ -0,0 +1,11 @@
+@testable import Kickstarter_Framework
+import XCTest
+
+final class CreatePasswordViewControllerTests: XCTestCase {
```

Comment:
We should probably test this just to be sure controller containment doesn't break in the future.